engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/content-host-register.controller.js

```diff
@@ -30,16 +30,23 @@ angular.module('Bastion.content-hosts').controller('ContentHostRegisterControlle
     function ($scope, $location, Capsule, Organization, CurrentOrganization, BastionConfig) {
 
         $scope.organization = Organization.get({id: CurrentOrganization});
-        $scope.baseURL = 'http://' + $location.host();
         $scope.consumerCertRPM = BastionConfig.consumerCertRPM;
+        $scope.katelloHostname = $location.host();
 
         $scope.capsules = Capsule.queryUnpaged(function (data) {
-            $scope.selectedCapsule = data.results[0];
+            var defaultCapsule = _.filter(data.results, function (result) {
+                var featureNames = _.pluck(result.features, 'name');
+                return _.contains(featureNames, 'Pulp');
+            });
+
+            $scope.noCapsulesFound = _.isEmpty(data.results);
```

Comment:
Can we initialize this to false outside of the async call here? Right now if I have no capsules, I will see the empty dropdown and then all of a sudden it will disappear.